### DISCUSSION TITLE
Auto merge button causes error toast

### DISCUSSION THREAD

**Discussion by 0xdef1cafe** [community/community#45912] [Mobile]
Enabling auto merge on a PR from the mobile app almost always shows an error toast, but auto merge is correctly enabled. Refreshing the UI then shows correct auto merge status. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.97.0 (89518966)
Device: iPhone 13 Pro (iOS 16.3)
</details>

**Comment by alcere** [comment]
Hi @0xdef1cafe, thanks for reporting. I have filled a bug, and the team will evaluate the fix. Please, stay tuned for updates!

**Comment by HenSquared** [comment]
Hi thanks for reporting this. 

We've recently merged a fix for this. It will be available in the app store release next Tuesday. 

The bug impacted users using 'squash' or 'rebase' enable auto merge option. As a workaround, users can ignore the error and refresh the PR to get the correct state.
